tests/multi-select.spec.ts

```diff
@@ -0,0 +1,254 @@
+import { test, expect } from '@playwright/test';
+import { waitForCustomEvent, waitForCustomEventWithParam } from './lib/helper.ts';
+
+test.beforeEach(async ({page}) => {
+    await page.goto('/multi-select-test.html')
+})
+
+test('Given a default multiselect, test options, labels and are visible and selecting, and clearing an options should correctly set pills', async ({page}) => {
```

Comment:
[nit] wording is a little confusing, maybe 
"Given a default multiselect, shows test options and labels, and sets pills correctly after selecting and clearing options"

Reply:
would add in the description that this test is also checking for correctly fired events